catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/cache/solr/impl/CachingFederationStrategyTest.java

```diff
@@ -738,9 +799,18 @@ public void testValidCollectionWithNullSource() throws Exception {
     strategy.federate(sources, fedQueryRequest);
   }
 
+  private Source getMockSource() throws UnsupportedQueryException {
+    Source mockSource = mock(Source.class);
+    when(mockSource.getId()).thenReturn("1");
```

Comment:
✏️ Source ids are supposed to be unique. CachingFederationStrategy doesn't really care, but it's not accurate to how the system is supposed to behave. I'd prefer to mock normal behavior by making these ids unique within the scope of each unit test.